netify_informatics.rst

```diff
@@ -4,100 +4,77 @@
 Netify Informatics
 ===================
 
-`Netify Informatics <https://www.netify.ai/products/netify-informatics>`_ is a third-party cloud service that utilizes analytics and AI to convert local DPI metadata obtained from NethSecurity into high-level network intelligence and visibility. The solution provides insights into various aspects of network activity, including:
+`Netify Informatics <https://www.netify.ai/products/netify-informatics>`_ is a third-party cloud service that utilizes analytics and AI to convert
+local DPI metadata obtained from NethSecurity into high-level network intelligence.
+The solution provides insights into various aspects of network activity, including:
 
-*   Device Discovery
-*   Geolocation
-*   Bandwidth Analysis
-*   Risk and Reputation Analysis
-*   Audit and Forensics
-*   Regulatory Compliance
+* `Device Discovery <https://www.netify.ai/products/netify-informatics/device-discovery>`_
+* `Bandwidth Monitoring <https://www.netify.ai/products/netify-informatics/bandwidth-monitoring>`_
+* `Risk and Reputation Analysis <https://www.netify.ai/products/netify-informatics/risk-and-reputation>`_
+* `Regulatory Compliance <https://www.netify.ai/products/netify-informatics/regulatory-compliance>`_
+* Geolocation
+* Audit and Forensics
 
-Here are some examples of what Netify Informatics can do:
+The service receives data from netifyd, the NethSecurity DPI engine which is enabled by default on the firewall.
 
+You can try the service for free for 7 day. 
+After this period, you can choose the plan that best fits your needs.
 
-`Bandwidth Monitoring <https://www.netify.ai/products/netify-informatics/bandwidth-monitoring>`_
-
-`Device Discovery <https://www.netify.ai/products/netify-informatics/device-discovery>`_
-
-`Risk and Reputation Analysis <https://www.netify.ai/products/netify-informatics/risk-and-reputation>`_
-
+See `Netify Informatics Pricing <https://www.netify.ai/products/netify-informatics/pricing>`_ and `Netify Informatics FAQ <https://www.netify.ai/resources/faq>`_ for more information.
 
 Before getting started
 ======================
 
-Here is some fundamental information to better manage your installations on Netify Informatics.
-You can granularly manage different customers, different locations of the same customer, and even different firewalls within the same location.The platform is organized with these elements.
+Make sure to create an account on the Netify Informatics website, you can try the service for free for 7 days.
```

Comment:
This "free trial" sentence is a repetition of a few lines above.

Reply:
I know, I want to tell it again: repetita iuvant!